packages/clerk-js/src/ui/components/SignUp/SignUpStart.tsx

```diff
@@ -103,14 +103,15 @@ function _SignUpStart(): JSX.Element {
   const hasTicket = !!formState.ticket.value;
   const hasEmail = !!formState.emailAddress.value;
   const isProgressiveSignUp = userSettings.signUp.progressive;
+  const legalConsentEnabled = userSettings.signUp.legal_consent_enabled;
```

Comment:
Should we rename this to isLegalConsentEnabled ?

Reply:
Yeah we can, so it can match the above, good suggestion

Reply:
Done!